### PR TITLE
Edits by GE for adding readme descriptions in main page and in DLICV sub-folder

### PR DESCRIPTION
**Pull request by gurayerus**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,31 @@
...
+## Segmentation Tasks <a name="seg_tasks"/>
+
+The table below lists models pre-trained for segmentation tasks that are part of this model repository. This table will evolve with time, as new models for new or existing tasks will be added. Models are tagged using a model name and a version number for reproducibility.
+
+|Task Name |Model Name |Date Upload |Description |File Name |
+|-|-|-|-|-|
+|<b>bmask</b>|deepmrseg_brain_v1.1|07/2021|A model trained for segmenting a brain mask|deepmrseg_bmask_v0.0.zip|
+|<b>wmlesion</b>|deepmrseg_wml_v1.1|07/2021|A model trained for segmenting white matter lesions|deepmrseg_wmlesion_v0.0.zip|
+|<b>hippocampus</b>|deepmrseg_hippo_v2.1|07/2021|A model trained for segmenting hippocampus sub-parts|deepmrseg_hippocampus_v0.0.zip|
+|<b>dlicv</b>|deepmrseg_dlicv_v1.1|07/2021|A model trained for segmenting intra-cranial volume|[deepmrseg_dlicv_v1.1.zip](https://github.com/gurayerus/DeepMRSeg-Models/blob/main/models/DLICV/DeepMRSeg_DLICV_v1.0.zip)|
```
Comment (ashishsingh18) [inline README.md]: Inconsistent Model Name, File Name and actual file in repo. The version should be *_v1.0*

Comment (ashishsingh18) [reply README.md]: Also, the url should point to upstream and not to a fork.

File: models/DLICV/README.md [hunk 2]
```diff
@@ -1 +1,49 @@
...
+deepmrseg_apply --task dlicv --inImg subj1_T1.nii.gz --outImg subj1_T1_DLICV.nii.gz
+
+# Batch processing of multiple subjects using a subject list
+#   User provides a csv file with columns: ID,InputT1,OutputImage
+deepmrseg_apply --task dlicv --sList subjectList.csv
+
+# Batch processing of multiple subjects in input folder 
+#   Testing is applied individually to all images with the given suffix in the input folder
+deepmrseg_apply --task dlicv --inDir myindir --outDir myoutdir --inSuff _T1.nii.gz --outSuff _DLICV.nii.gz
+```
```
Comment (ashishsingh18) [inline models/DLICV/README.md]: This should be moved to DeepMRSeg repo. Here we should just give a link to that. Something like:

For usage, please see the [user manual](https://github.com/CBICA/DeepMRSeg) from DeepMRSeg repository.

File: models/DLICV/README.md [hunk 3]
```diff
@@ -1 +1,49 @@
...
+
+This causes a consistent bias if the brain mask is used for estimating the ICV: It under-estimates it when a subject has high global cortical atrophy, e.g. with aging, and over-estimates it in younger subjects, as the brain mask is more prone to include dura and other non-brain tissues in areas difficult to segment.
+
+T2-weighted images provide a better contrast for delineating the cortical CSF. However, high resolution T2 images may not be always available. Also, multi-modal techniques bring increased complexity, as they require additional preprocessing steps. 
+
+The main motivation for DLICV was based on a simple hypothesis:
+
+*"The actual boundary of the ICV, which is best differentiated on T2 images, can be learned ***using only raw T1 images as input***, if a model is intensively trained using accurate ground-truth segmentations obtained from T2 images."*
+
+Validation experiments have demonstrated that the proposed strategy resulted in accurate ICV masks. The final model was trained on a very rich dataset with n=450 subjects from 15 studies, and their carefully curated/verified reference labels [TODO: ref to paper]
```
Comment (ashishsingh18) [inline models/DLICV/README.md]: Do we need to give these details here? These might be better going in the paper? We can keep a shorter version here. Let's discuss tomorrow.

**Review by ashishsingh18 — CHANGES_REQUESTED** [review]
Suggesting some modifications. Let's discuss tomorrow.
Perhaps we should also add model parameters, model generation procedure and data description.

File: README.md (hunk 1 above)
Comment (gurayerus) [reply README.md]: All fixed

File: models/DLICV/README.md (hunk 2 above)
Comment (gurayerus) [reply models/DLICV/README.md]: Corrected

File: models/DLICV/README.md (hunk 3 above)
Comment (gurayerus) [reply models/DLICV/README.md]: I agree, but kept it here for now.

**Review by ashishsingh18 — APPROVED** [review]
looks fine to me. We can keep updating it.

**Pull request merged**